DocumentDataSubmission/src/main/resources/wsdl/EntityXDRSecured.wsdl

```diff
@@ -27,9 +25,6 @@
             <xsd:import namespace="urn:gov:hhs:fha:nhinc:gateway:samltokendata" schemaLocation="classpath:schemas/nhinc/gateway/SamlTokenData.xsd" />
             <xsd:import namespace="urn:gov:hhs:fha:nhinc:common:nhinccommonentity" schemaLocation="classpath:schemas/nhinc/common/NhincCommonEntity.xsd" />
         </xsd:schema>
-        <!-- Importing all schemas within one seems to confuse some parsers and code-generation tools (and may be invalid) 
-            <xsd:schema elementFormDefault="qualified"> <xsd:import namespace="urn:oasis:names:tc:ebxml-regrep:xsd:rs:3.0" schemaLocation="../schema/ebRS/rs.xsd"/> 
-            <xsd:import namespace="urn:ihe:iti:xds-b:2007" schemaLocation="../schema/IHE/XDS.b_DocumentRepository.xsd"/> </xsd:schema> -->
     </types>
     <message name="RegisterDocumentSet-b_Message">
         <documentation>Register Document Set</documentation>
```

Comment:
Fix the soap address to https://localhost:${HttpsDefaultPort}/Gateway/DocumentDataSubmission/1_1/EntityService/EntityDocDataSubmissionSecured"